### PR TITLE
pr

### PR DESCRIPTION
**Pull request by alannatai**

### CODE REVIEW

File: src/components/Contact/Contact.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Contact.css';
+
+function Contact(props) {
+	const data = props.contact;
+	const location = props.contact.location;
```
Comment (foxfriends) [inline src/components/Contact/Contact.jsx:6]: I think using destructuring for `props` will make things somewhat clearer. This way you have a nice list of exactly what props are supported, and also don't have to write `props.__` everywhere in the component.

```suggestion
function Contact({ 
  contact: { 
    email, 
    phone, 
    website, 
    location: { address, postalCode, city, region, countryCode },
  },
}) {
```

File: src/components/Body/Body.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.Body {
+	display: grid;
+	grid-template-columns: 30% 60%;
+	grid-template-areas: 'aside main';
+	grid-column-gap: 50px;
```
Comment (foxfriends) [inline src/components/Body/Body.css:5]: Nice use of grid!

File: src/components/BasicSection/BasicSection.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './BasicSection.css';
+
+function BasicSection(props) {
```
Comment (monarchwadia) [inline src/components/BasicSection/BasicSection.jsx:4]: This is only used in 1 place, and is a tiny file. No need to have a separate component.

File: src/components/Body/Body.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import Aside from '../Aside/Aside';
+import Main from '../Main/Main';
+
+import './Body.css';
+
+function Body(props) {
+	return (
+		<div className="Body">
+			<Aside contact={props.basics} />
+			<Main summary={props.basics.summary} experience={props.experience} />
```
Comment (monarchwadia) [inline src/components/Body/Body.jsx:11]: Splitting both Aside as well as Main is not necessary. Files are very small.

**Pull request closed without merging**